tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md

```diff
@@ -0,0 +1,159 @@
+---
+id: tutorial-install-ubuntu-desktop
+summary: Temukan betapa mudahnya menginstal desktop Ubuntu ke komputer laptop atau PC Anda, dari keduanya
+DVD atau flash drive USB.
+categories: desktop
+tags: tutorial,installation,ubuntu,desktop,Ubuntu 16.04 LTS
+difficulty: 2
+status: Published
+published: 2017-07-06
+author: Canonical Web Team <webteam@canonical.com>
+translator: Reza Juliandri
+
+---
+
+# Memasang Ubuntu desktop
+
+## Pendahuluan
+Durasi: 0:05
```

Comment:
Please change "Duration" back to english, this won't be display to users, but used internally by the tutorials engine, thanks!